src/app/visual/editable-grid/editable-grid.component.scss

```diff
@@ -0,0 +1,29 @@
+@import '~@skyux/theme/scss/mixins';
+
+.complete i {
+  color: $sky-highlight-color-success;
+}
+
+.current i {
+  color: $sky-highlight-color-warning;
+}
+
+.behind i {
+  color: $sky-highlight-color-danger;
+}
+
+.status {
+  padding: 0 2px;
+}
+
+.green {
```

Comment:
Wrap these with `::ng-deep` to avoid having to turn off view encapsulation on the component.